viv-wallet-app-vue/src/main.ts

```diff
@@ -4,10 +4,11 @@ import router from "./router";
 import BootstrapVue from "bootstrap-vue";
 import "bootstrap/dist/css/bootstrap.css";
 import "bootstrap-vue/dist/bootstrap-vue.css";
+import Banner from "./components/Banner.vue";
 
 Vue.config.productionTip = false;
 Vue.use(BootstrapVue);
-
+Vue.component("Banner", Banner);
```

Comment:
You don't need to register your component globally.
Just register it locally where you use it